Documents/Documents/Code/Controllers/Rooms/RoomInfoFlow/RoomSharingNetworkService.swift

```diff
@@ -91,6 +91,22 @@ final class RoomSharingNetworkService: RoomSharingNetworkServiceProtocol {
             completion(.success(responce))
         }
     }
+    
+    func duplicateRoom(room: ASCFolder, completion: @escaping (Result<ASCFolder, Error>) -> Void) {
+        guard let roomId = Int(room.id) else { return }
```

Comment:
there is could be a string not Int only, change the model

Reply:
✅